### ISSUE TITLE
Prevent block comments

### ISSUE THREAD

**Issue by emretoptanci** [editorconfig/editorconfig#486]
"Block comments" are evil. They are especially used to disable a large part of code easily but when you do that it becomes hard to read the code. The difference is not obvious in the Pull Request. Just two lines of difference which in fact disables maybe hundreds of lines of code. You inspect lines and lines of code just to realize that that code does not run because it is put inside a block comment. 
Line comments on the other hand are angels. Every affected line of code is changed when a line comment is applied so you never miss that line in the Pull Request.
A new configuration option should be introduced to "Prevent block comments" or "automatically turn block comments into line comments".

**Comment by cxw42** [comment]
I agree that line comments are preferable.  However, finding block comments requires semantic analysis, in the general case.  EditorConfig supports editors that do not have analysis capability.  I am therefore not sure if EditorConfig is the right place for this setting.  E.g., I would expect to see this in a linter config.
